backend/src/app.ts

```diff
@@ -69,21 +87,39 @@ app.post('/landlords', async (req, res) => {
   }
 });
 
+const isLandlord = (obj: LandlordWithId | ApartmentWithId): boolean => {
+  const keys = Object.keys(obj);
+  return keys.includes('contact');
+};
+
 app.get('/reviews', async (req, res) => {
   try {
     const query = req.query.q as string;
     const landlordDocs = (await landlordCollection.get()).docs;
-    const landlords: Landlord[] = landlordDocs.map((landlord) => landlord.data() as Landlord);
+    // eslint-disable-next-line
```

Comment:
you can easily fix these eslint errors by removing the curly brackets. If you omit the `return` in an arrow function in JS, and your function is just a value, JS assumes that you're returning that value on each iteration. For this example, it'd be

```
 const landlords: LandlordWithId[] = landlordDocs.map((landlord) => (
      { id: landlord.id, ...landlord.data() } as LandlordWithId
 ));
```